docs/src/lib/sets/SparsePolynomialZonotope.md

```diff
@@ -124,6 +124,7 @@ Inherited from [`LazySet`](@ref):
 * [`rationalize`](@ref rationalize(::LazySet))
 * [`rectify`](@ref rectify(::LazySet))
 * [`reflect`](@ref reflect(::LazySet))
+* [`scale`](@ref scale(::Real, ::LazySet))
```

Comment:
This should go to the block `Undocumented implementations:`, and the entry in line 137 should be removed.